### DISCUSSION TITLE
update whatsapp profile name

### DISCUSSION THREAD

**Discussion by ridhodo** [tulir/whatsmeow#333] [WhatsApp protocol Q&A]
Hi, I already try change name account whatsapp use this function

`WAClient.SetGroupName(jid, imcenter.AccountName.String)`, and use EmptyJID. 

Have anyone success to update whatsapp account name ? please tell me the clue.
Thankyou

**Comment by jeffersonsc** [comment]
Hello try the code above

```
err := client.SendAppState(appstate.BuildSettingPushName("New Name"))
	if err != nil {
		return err
	}
```
